src/app/components/icons/CrossIcon.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+const propTypes = {
+    className: PropTypes.string,
+    ariaLabel: PropTypes.string,
+    fill: PropTypes.string
+};
+
+const defaultProps = {
+    className: "svg-icon--hide-notification",
+    ariaLabel: "Hide icon",
+    fill: "#FFFFFF"
+};
+
+class CrossIcon extends Component {
```

Comment:
would be good to use functional component instead of class. you can take a look here for inspiration :-) also try to use `props.variant ` not `props.className` as this might be confusing 
https://github.com/ONSdigital/florence/pull/602/files#diff-abc9dd8fd0dd4bab595b99e0b416dd53c1efae9fad4542648deebd82ec848662R12